src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -61,17 +62,36 @@ type Props = TaskFormProps & TaskFormConsumerProps & InjectIntlProvidedProps;
 type TaskFormFieldName = 'taskName' | 'taskAssignees' | 'taskDueDate';
 
 type State = {|
+    addedAssignees: SelectorItems,
     approvers: SelectorItems,
     dueDate?: ?Date,
     formValidityState: { [key: TaskFormFieldName]: ?{ code: string, message: string } },
     id: string,
     isLoading: boolean,
     isValid: ?boolean,
     message: string,
+    removedAssignees: Array<TaskCollabAssignee>,
 |};
 
+function getUsersFromTask(approvers: Array<TaskCollabAssignee>): SelectorItems {
```

Comment:
This can be in its own file, something like utils.js locally

Reply:
We have a future optimization to possibly align the types across components to avoid this conversion, but put that on the back burner for now. I'm a fan of component util.s in general, tho. easier to unit test